charts/cluster-agent/values.yaml

```diff
@@ -1,19 +1,19 @@
 # IMPORTANT: Specify your Epsagon Token here.
 # epsagonToken -- Set the Epsagon token of your account
 epsagonToken: ""
+
+# -- Set a unique cluster name to allow scoping hosts and Cluster Checks easily
```

Comment:
needed?

Reply:
You want to change the description? We can say something like:
"Set a unique cluster name to allow multiple clusters to integrate with epsagon, and easily identify where the resources are coming from"